src/main/java/org/example/homework3/DivisionOperation.java

```diff
@@ -0,0 +1,14 @@
+package org.example.homework3;
+
+public class DivisionOperation {
+    public static void main(String[] args) {
+        int dividend = 150;
+        int divisor = 20;
+        int integerPart = dividend/divisor;
```

Comment:
minor, please format code
```suggestion
        int integerPart = dividend / divisor;
```